src/library/Uno.Themes.WinUI.Markup/Themes/Slider.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class Slider
+		{
+			public static class Resources
+			{
+				public static class Typography
+				{
+					[ResourceKeyDefinition(typeof(FontFamily), "SliderFontFamily")]
+					public static ResourceValue<FontFamily> FontFamily => new("SliderFontFamily", true);
+
+					[ResourceKeyDefinition(typeof(double), "SliderFontSize")]
+					public static ResourceValue<double> FontSize => new("SliderFontSize", true);
+				}
+
+				public static class Track
+				{
+					public static class Fill
+					{
+						[ResourceKeyDefinition(typeof(Brush), "SliderTrackFill")]
+						public static ResourceValue<Brush> Default => new("SliderTrackFill", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "SliderTrackFillDisabled")]
+						public static ResourceValue<Brush> Disabled => new("SliderTrackFillDisabled", true);
+					}
+
+					public static class ValueFill
+					{
+						[ResourceKeyDefinition(typeof(Brush), "SliderTrackFill")]
+						public static ResourceValue<Brush> Default => new("SliderTrackFill", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "SliderTrackFillDisabled")]
+						public static ResourceValue<Brush> Disabled => new("SliderTrackFillDisabled", true);
+					}
+
+					public static class CornerRadius
```

Comment:
Doesn't need to have a Default, can just be `Slider.Resources.Track.CornerRadius` as a direct property

Reply:
Fixed.